src/overlays/ovl_i3/C2290.c

```diff
@@ -54,7 +54,11 @@
 
 #pragma GLOBAL_ASM("asm/us/rev0/nonmatchings/overlays/ovl_i3/C2290/func_i3_80134548.s")
 
-#pragma GLOBAL_ASM("asm/us/rev0/nonmatchings/overlays/ovl_i3/C2290/func_i3_80134598.s")
+extern s32 D_8010D768;
+
+void func_i3_80134598(void) {
+    D_8010D768 = 0;
```

Comment:
```suggestion
    D_i2_8010D768 = 0;
```

This may require adding it to symbol_addrs_overlays.txt, lmk if you have any issues doing that